_partials/ssh_key.md

```diff
@@ -25,42 +25,15 @@ Now, you will give your **public** key to GitHub.
 In your terminal copy-paste the following command:
 
 ```bash
-cat ~/.ssh/id_ed25519.pub
+gh auth refresh -s write:public_key
 ```
 
-It will prompt on the screen the content of the `id_ed25519.pub` file.
+It will prompt a one time code (####-####) on the screen. Copy it and press `ENTER`, then paste the code in your browser and follow the instructions to **Authorize github**.
```

Comment:
```suggestion
It will prompt a one time code (####-####) on the screen. Copy it and press `ENTER`, then paste the code in your browser and follow the instructions to **Authorize GitHub**.
```